### PR TITLE
deploy and configure a parse server

### PR DESCRIPTION
**Pull request by shirleymango**
- created Parse database on Back4App
- added application ID and client key to Info.plist
- added Info.plist to .gitignore file
parse database

### CODE REVIEW

**Review by ebeweber-fb — COMMENTED** [review]
Good use of .gitignore!

File: metau-capstone/Info.plist [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>app_ID</key>
+	<string>ZuKS1LMhQrXcAWcQZxcHVPLYmACt7Ch5MrWCjH9v</string>
+	<key>client_Key</key>
```
Comment (zenangao-fb) [inline metau-capstone/Info.plist:7]: change `client_Key` to `parse_client_key` & `app_ID` to `parse_ app_ID`

File: metau-capstone/LoginViewController.m [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            NSLog(@"User logged in successfully");
+            
+            // display view controller that needs to shown after successful login
+        }
+    }];
+}
+
+- (IBAction)registerUser:(id)sender {
+    // initialize a user object
+    PFUser *newUser = [PFUser user];
```
Comment (zenangao-fb) [inline metau-capstone/LoginViewController.m:38]: nit: const

eg.
```
PFUser *const newUser = [PFUser user];

File: metau-capstone/LoginViewController.m [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    
+    // set user properties
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
```
Comment (zenangao-fb) [inline metau-capstone/LoginViewController.m:42]: things to think about, what about input validation & empty character truncation?
- english character only?
- truncate empty character before and after the input?
- double passwords with confirmation capability
- hide as passwords as user type
- option to view password

File: metau-capstone/LoginViewController.m (hunk 2 above)
Comment (shirleymango) [reply metau-capstone/LoginViewController.m:38]: Okay! Why do I want to use const here?

Comment (zenangao-fb) [reply metau-capstone/LoginViewController.m:38]: This is a coding standard, if any local variable is intended not to be reassigned, annotate it with const would safeguard it to be unintentionally reassigned later on

**Pull request merged**